packages/opds-web-client/src/components/Root.tsx

```diff
@@ -267,7 +269,6 @@ export class Root extends React.Component<RootProps, RootState> {
                     collection={this.collectionDataWithLoans()}
                     fetchPage={this.props.fetchPage}
                     isFetchingCollection={this.props.isFetchingCollection}
-                    isFetchingBook={this.props.isFetchingBook}
```

Comment:
what happened here?

Reply:
This wasn't used in the app, but now I'm thinking it's for an inherited use-case?

Reply:
I'm not sure about this one - it seems like it should be needed though, to show the loading message if you go directly to a book with no collection.